docs/apis/streaming/fault_tolerance.md

```diff
@@ -176,6 +176,11 @@ state updates) of Flink coupled with bundled sinks:
         <td></td>
     </tr>
     <tr>
+        <td>Cassandra sink</td>
+        <td>exactly once</td>
```

Comment:
which is also not true. a flink failure while data is being written to cassandra will cause duplicates. you can only say this if writing the data to the final table is completely handled by cassandra (for example by writing into a temporary table, exporting it to csv and importing into the target table; the only way for duplicates is if cassandra fails while importing).